Assets/Scripts/Models/Time/WorldTime.cs

```diff
@@ -294,13 +315,13 @@ public int Year
 
     #region IFormattable implementation
 
-    string IFormattable.ToString(string format, IFormatProvider provider = null)
+    string IFormattable.ToString(string format, IFormatProvider provider)
```

Comment:
This was here intentionally so that it can be used without having to pass a provider (even if it is null), to allow for cleaner usage (imo intentionally passing nulls looks bad, and is pointless when it can be done otherwise like this)... 

Unless, of course, IFormattable does this on its own behind the scenes...